bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/report/job/AnalyzeJob.java

```diff
@@ -39,78 +46,98 @@ private AnalyzeJob(String name) {
      * @param project The eclipse project that's will be analyzed.
      */
     public AnalyzeJob(IProject project) {
+        this(project, null);
+    }
+
+    /**
+     * Constructor for making an analyze job, for one file;
+     * 
+     * @param project
+     *            The eclipse project that's will be analyzed.
+     */
+    public AnalyzeJob(IProject project, Path file) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected @param tag for 'file'.](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641621)